### PR TITLE
[Docs][CDCSDK] Update docs Debezium Connector for YugabyteDB

### PR DESCRIPTION
**Pull request by vaibhav-yb**
This PR has the diff for the following:
* Addition of a new config parameter description `tombstones.on.delete`
* Addition of a new GFlag meant to enable/allow the TRUNCATE operation on tables on which CDCSDK streams are configured

@netlify /preview/explore/change-data-capture/debezium-connector-yugabytedb/

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 1]
```diff
@@ -842,9 +842,9 @@ To learn about CDC, see [Change data capture (CDC)](../../../architecture/docdb-
 
   Default: `102400`
 
-##### --stream_truncate_record
+##### --enable_delete_truncate_cdcsdk_table
 
-  Enable streaming of TRUNCATE record for a table on which CDC is active.
+  By default, TRUNCATE commands on tables on which CDCSDK stream is active will fail. Changing the value of this flag from `false` to `true` will enable truncating the tables part of the CDCSDK stream.
```
Comment (polarweasel) [inline docs/content/preview/reference/configuration/yb-tserver.md:847]: Do you want to make this change in /stable as well, or was this new in 2.15?

**Comment by polarweasel** [comment]
@vaibhav-yb I've made a bunch of edits to preview/.../debezium-connector-yugabytedb.md.

**Please review commit** [`275083f` (#14261)](https://github.com/yugabyte/yugabyte-db/pull/14261/commits/275083f9ebb5a95f72909bb1fe0558c5e70e65fb) and let me know if there's anything wrong. If not, let's do the same in /stable and then we should be good to merge.

File: .github/vale-styles/Yugabyte/spelling-exceptions.txt [hunk 2]
```diff
@@ -46,6 +46,7 @@ autoscaled
 autoscaler
 autoscales
 autoscaling
+Avro
```
Comment (polarweasel) [inline .github/vale-styles/Yugabyte/spelling-exceptions.txt:49]: BTW, this file is for the linter and spell checker we use :)

File: docs/content/preview/reference/configuration/yb-tserver.md (hunk 1 above)
Comment (vaibhav-yb) [reply docs/content/preview/reference/configuration/yb-tserver.md]: This change is new in 2.15 itself so we can keep it there only.

**Comment by vaibhav-yb** [comment]
Hi @polarweasel The commit you made looks fine to me. One question I have is that there are certain changes in the title of "warning" and "notes" - has the structure changed recently for SEO purposes?

**Pull request merged**